SingularityExecutor/src/main/java/com/hubspot/singularity/executor/models/LogrotateTemplateContext.java

```diff
@@ -69,10 +71,30 @@ public String getCompressExt() {
   }
 
   /**
-   * Extra files for logrotate to rotate. If these do not exist logrotate will continue without error.
+   * Extra files for logrotate to rotate (non-hourly). If these do not exist logrotate will continue without error.
    * @return filenames to rotate.
    */
   public List<LogrotateAdditionalFile> getExtrasFiles() {
+    return getAllExtraFiles().stream()
+        .filter(p -> !p.getLogrotateFrequencyOverride().equals(SingularityExecutorLogrotateFrequency.HOURLY.getLogrotateValue()))
+        .collect(Collectors.toList());
+  }
+
+  /**
+   * Extra files for logrotate to rotate hourly. If these do not exist logrotate will continue without error.
+   * @return filenames to rotate.
+   */
+  public List<LogrotateAdditionalFile> getExtrasFilesHourly() {
+    return getAllExtraFiles().stream()
+        .filter(p -> p.getLogrotateFrequencyOverride().equals(SingularityExecutorLogrotateFrequency.HOURLY.getLogrotateValue()))
+        .collect(Collectors.toList());
+  }
+
+  public boolean getIsGlobalLogrotateHourly() {
```

Comment:
For booleans, the standard is to use the `is` prefix by itself (`isGlobalLogrotateHourly`). A number of libraries like reflections (and the the handlebars templater in this case) go by that standard.